src/test/scala/nl/knaw/dans/easy/ps/AddEmdToDatasetIngestDirSpec.scala

```diff
@@ -47,6 +47,13 @@ class AddEmdToDatasetIngestDirSpec extends UnitSpec {
     }) checkPreconditionsShould be(a[Success[_]])
   }
 
+it should "fail if person creators and organization-only creators are mixed" in {
```

Comment:
It should *not* fail. That was my point. Use the Web-UI as a reference of what is permissible.